python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -111,9 +108,44 @@ def __init__(
         # Task for self.read_loop
         self.read_loop_task = salobj.make_done_future()
 
-        # Most recent Rotator Application telemetry sample,
-        # or None if camera cable wrap not tracking.
-        self.prev_rot_application = None
+        # Most recent Rotator Application telemetry sample information,
+        # or None if no information from the rotator was ever received.
+        self.rotator_time = None
+        self.rotator_position = None
+        self.rotator_velocity = None
+        self.rotator_demand = None
+
+        # Event to indicate that the rotator position was updated. It will be
+        # set when callback is called.
+        self.rotator_position_updated = asyncio.Event()
+
+        # Most recent Camera Cable Wrap telemetry sample information, or None
+        # if no information from CCW was ever been received
+        self.ccw_angle = None
+        self.ccw_velocity = None
+        self.ccw_time = None
+
+        # Demand position for the CCW. None is CCW tracking is off.
+        self.ccw_demand_position = None
+        self.ccw_demand_velocity = None
+        self.catch_up_mode = False
+
+        # CCW - Rotator synchronization limits.
+        # The goto limit is the distance between the CCW and the Rotator demand
+        # position after which the CCW will blindly go to the demand, as long
+        # as the distance between the ccw and rot are less then half the max
+        # limit.
+        # The max limit is the maximum distance allowed between CCW and
+        # Rotator.
+        # The slew limit is the distance between CCW - Rotator where CCW will
+        # enter "slew" mode, and start following the position of the Rotator.
+        # The track limit is the distance between CCW - Rotator at which they
+        # are considered in synchronization and CCW will follow the Rotator
+        # demand.
+        self.ccw_rot_sync_limit_goto = 7.5
+        self.ccw_rot_sync_limit_max = 2.0
+        self.ccw_rot_sync_limit_slew = 0.25
+        self.ccw_rot_sync_limit_track = 0.125
```

Comment:
Once we test this with real hardware and demonstrate that it works (and I think it quite likely it will) then we should make these values configuration parameters. Could you file a ticket for that? Feel free to assign it to me.